package.json

```diff
@@ -19,7 +19,8 @@
     "lint": "standard"
   },
   "dependencies": {
-    "invariant": "2.2.4"
+    "invariant": "2.2.4",
+    "lodash": "^4.17.21"
```

Comment:
If you can remove the `^` character, this should be good to go

Reply:
@merceyz 👆

Reply:
I don't get why you want that but sure

Reply:
Internally we use specific version numbers for security purposes. It's something we're hoping to change in the future once we can convince the right teams, but for the time being this is the approach we have to take. Thanks for making this change!